sdcm/sct_provision/user_data_objects/scylla.py

```diff
@@ -26,9 +28,16 @@ def is_applicable(self) -> bool:
     @property
     def scylla_machine_image_json(self) -> str:
         """Specifies configuration file accepted by scylla-machine-image service"""
-        return dedent("""{
-                     "scylla_yaml": {
-                         "experimental": true
-                     },
-                     "start_scylla_on_first_boot": false
-                }""")
+        if (self.params.get("data_volume_disk_num") or 0) > 0:
```

Comment:
```suggestion
        if (self.params.get("data_volume_disk_num", 0) > 0:
```

Reply:
`params.get()` method does not accept defaults... :(